src/sparql/icd11foundation-icd10-xrefs.ru

```diff
@@ -0,0 +1,17 @@
+# TODO: temp: FYI I originally copy/based template from ordo-replace-annotation-based-mappings.ru
+
+# todo: do i need all these namespaces?
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+prefix sssom: <https://w3id.org/sssom/>
+
+
+DELETE {
+  ?cls http://id.who.int/icd/schema/icd10code ?xref .
+} INSERT {
+  # TODO: add prefix declaration for ICD10WHO or wrap in URI() around CONCAT()
+  ?cls oboInOwl:hasDbXref ICD10WHO?xref CONCAT("ICD10WHO", REPLACE(?xref, "NEED_EXAMPLE_SHOWING_URI_STEM", ""))
```

Comment:
## `icd10code`
- [ ] Advise @matentzn / @twhetzel

I started writing a SPARQL query to remap `icd10code` annotation properties to `oboInOwl:hasDbXref ICD10WHO:<CODE>`. However I just realized that there exists only a property declaration for `icd10code`, no usages.

Decisions:
1. What to do with the annotation prop declaration?
  a. remove
  b. keep
2. What to do with my existing SPARQL file?
  a. Delete it since it's not needed
  b. Keep it for the future in case we need it; keep it blank
  c. Keep it for the future in case we need it; leave it as is: unfinished w/ TODO's